eogrow/utils/types.py

```diff
@@ -12,3 +12,5 @@
 
 Feature = Tuple[FeatureType, str]
 FeatureSpec = Union[Tuple[FeatureType, str], FeatureType]
+
+BoolOrAuto = Union[Literal["auto"], bool]
```

Comment:
do you suppose such a type will occur often?

Reply:
I think the type is general enough that it can occur more often in the future for various pipeline parameters. This way it is also clearer that the type that is used in pipeline schema and in the utility function is the same. Also in general we should try to avoid code duplications of complicated type definitions (so far, I also wasn't paying much attention about that myself).